### DISCUSSION TITLE
Where to put skins and themes I downloaded from the official website?

### DISCUSSION THREAD

**Discussion by zShxck720** [smplayer-dev/smplayer#860] [Q&A]
I'm using Debian and I installed the .deb package so I have to manually add skins and themes but I don't remember the path.
Could also be a REALLY good idea to write it on the website or inside the README file...

**Comment by smplayer-dev** [comment]
There are deb packages for the themes:
sudo apt-get install smplayer-themes smplayer-skins

If you want to install them manually, you can copy them to `~/.config/smplayer/themes`

**Reply by zShxck720** [reply]
Thank you
